### DISCUSSION TITLE
[New Component] [ElSplit] 希望新增组件：面板分割组件

### DISCUSSION THREAD

**Discussion by lijinzhou2016** [element-plus/element-plus#18727] [Feature Request]
Feature type: **`New Component`**

## Details

- New component name: `ElSplit`

### Features / 特性
Split a page into two resizable sections.

### Living examples / 已有的例子
https://arco.design/vue/component/split

### Additional comments

_(empty)_

**Comment by xbduck** [comment]
确实有用到

**Comment by namefhf** [comment]
https://github.com/antoniandre/splitpanes

**Reply by thinkasany** [reply]
例子 + 1
https://ant.design/components/splitter-cn

**Reply by elarbor** [reply]
你的例子打不开啊

**Comment by xbduck** [comment]
自动回复：您好，你的邮件已经收到，我会尽快拜读，祝你天天开心，万事如意！

**Comment by Dsaquel** [comment]
I can work on it soon.

**Comment by xbduck** [comment]
自动回复：您好，你的邮件已经收到，我会尽快拜读，祝你天天开心，万事如意！

**Comment by elarbor** [comment]
好巧，我也在找 [Splitter](https://vue-devui.github.io/components/splitter/) 组件。https://ask.csdn.net/questions/8178526
